src/server/api/routers/company.ts

```
@@ -66,4 +67,38 @@ export const companyRouter = createTRPCRouter({
       },
     })
   }),
+
+  getInvestmentByCompany: publicProcedure
+    .input(
+      z.object({
+        limit: z.number().min(1).max(100).nullish(),
+        companyId: z.string().nullish(),
```

Hmmm. Actually not sure about if `companyId` should be `nullish()` here, since this query relies on there being a `companyId` to search for.

---

Alternatively, we can throw a tRPC error if the `companyId` is not there (check [this](https://github.com/toriis-portal/toriis/blob/main/src/server/api/routers/esg.ts#L62) for instance).